pennylane_lightning_kokkos/src/simulator/StateVectorKokkos.hpp

```diff
@@ -77,7 +81,7 @@ template <class Precision> class StateVectorKokkos {
                      Kokkos::MemoryTraits<Kokkos::Unmanaged>>;
 
     StateVectorKokkos() = delete;
-    StateVectorKokkos(size_t num_qubits)
+    StateVectorKokkos(size_t num_qubits, Kokkos::InitArguments kokkos_args = {})
```

Comment:
Is a copy the best way to pass this in? What about using references?

Reply:
Yes, it will be more efficient to pass by reference. I could change for 
```
const Kokkos::InitArguments &kokkos_args = {}
```
`const` is necessary to set the default value to {}, I think. But that's alright since `kokkos_args` needs not be modified all the way from the Python layer.

Reply:
Feel free to add the update :)